modules/claims_api/spec/support/rswag_config.rb

```diff
@@ -115,15 +116,52 @@ def config # rubocop:disable Metrics/MethodLength
               name: :token,
               scheme: :bearer,
               bearer_format: :JWT
+            },
+            productionOauth: {
+              type: :oauth2,
+              description: 'This API uses OAuth 2 with the authorization code grant flow. [More info](https://developer.va.gov/explore/authorization?api=claims)',
+              flows: {
+                authorizationCode: {
+                  authorizationUrl: 'https://api.va.gov/oauth2/authorization',
+                  tokenUrl: 'https://api.va.gov/oauth2/token',
+                  scopes: {
+                    'claim.read': 'Retrieve claim data',
+                    'claim.write': 'Submit claim data'
+                  }
+                }
+              }
+            },
+            sandboxOauth: {
+              type: :oauth2,
+              description: 'This API uses OAuth 2 with the authorization code grant flow. [More info](https://developer.va.gov/explore/authorization?api=claims)',
+              flows: {
+                authorizationCode: {
+                  authorizationUrl: 'https://sandbox-api.va.gov/oauth2/authorization',
+                  tokenUrl: 'https://sandbox-api.va.gov/oauth2/token',
+                  scopes: {
+                    'claim.read': 'Retrieve claim data',
+                    'claim.write': 'Submit claim data'
+                  }
+                }
+              }
             }
           }
         },
         paths: {},
-        basePath: '/services/benefits/v2',
+        basePath: '/services/claims/v2',
```

Comment:
We sure this is correct?
The Lighthouse platform is configured in DEV to hit our V2 endpoints at `/services/benefits/v2/`.
[REF](https://github.com/department-of-veterans-affairs/lighthouse-platform-tools/blob/main/environments/dev/config/vets-api.yaml#L267)

(fwiw, it's been a while since I worked in V2, but I have old Postman configs that also are hitting the base path as `/services/benefits/v2/`)

Reply:
Oh I just re-read the Jira ticket and the first bullet of required changes says that `basePath` needs to be removed.  So I'm guessing this attribute shouldn't exist here at all anymore?

Reply:
I had issues with rswag tests failing that seemed related to this field, so even though it's not part of OAS v3, it seems rswag might still need it. I'll double check if that's actually the case and remove if I can. It might just be a variable reference that I need to update elsewhere

Reply:
Turns out this is a known [issue with rswag](https://github.com/rswag/rswag/issues/318) :( in the meantime, one of the suggestions put forth in that issue thread is to manually remove the `basePath` propertiy from the generated JSON, which might be a temporary solution, but tedious to do every time. I'll see if I can implement a workaround in `rswag.rake` to go in and remove that key, but thoughts on the manual approach for now?

Reply:
Wow, that really stinks.  😞 
The manual approach is prob fine for now but we should probably create a ticket so we can invest some time into trying to find an automated solution like you hinted at.  Because I know over time, we'll start to forget to come back and remove `basePath` from the generated file.